### ISSUE TITLE
fotmob_get_match_players

### ISSUE THREAD

**Issue by daanih85** [JaseZiv/worldfootballR#298]
Hi!

It seems like Fotmob have changed things around as scraping player stats now looks like this: 

<img width="872" alt="Skjermbilde 2023-07-03 kl  19 14 17" src="https://github.com/JaseZiv/worldfootballR/assets/116917476/4f174bfe-18e3-45dd-ae2d-991b143b5fcd">

Wondering if this is fixable?

---
```r
library(worldfootballR)
packageVersion("worldfootballR")
[1] ‘0.6.3.7’

fotmob_matches <- c(3609994, 3610132)
players <- fotmob_get_match_players(fotmob_matches)
```

**Comment by tonyelhabr** [comment]
Ah yes, that looks like a data change on their end. I'm not sure I'll get to it this weekend, but maybe next week.

**Comment by kidshiva** [comment]
Hi both,

Just checked the merge fix for fotmob_get_match_players and it looks like there is an issue with the data

after a certain point in the nesting, the players data doesnt change for the player. i have attached a screenshot of what I am seeing, but it seems to be repeating the data for three players at a time rather than giving the individual players data

![image](https://github.com/JaseZiv/worldfootballR/assets/46587882/535f2b7f-2937-48b6-86a4-e0effc1e382f)

Using worldfootballR_0.6.3.0010

**Comment by heartsstats** [comment]
Just found the same issue. Looks like there is an issue with players from the same team and having the same value in the position_row column having their stats values linked together in error.

**Comment by tonyelhabr** [comment]
being addressed with #303
